File: Applicationjava/myenvent/src/main/java/com/ly/myenvent/MainActivity.java
package com.ly.myenvent;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
    }

    public void btnClick(View view) {
        int id = view.getId();
        if (id == R.id.button1){
            Intent intent = new Intent(this, MainActivity2.class);
            startActivity(intent);
        }
    }
}

File: Applicationjava/mylifecycle/src/main/java/com/ly/mylifecycle/MainActivity.kt
package com.ly.mylifecycle

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.view.View

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)


    }

    fun btnClick(view: View) {
        val id = view.id
        if (id == R.id.button1){
            startActivity(Intent(this,MainActivity2::class.java))
        }else if(id == R.id.button2){
            startActivity(Intent(this,MainActivity3::class.java))
        }else if(id == R.id.button3){
            startActivity(Intent(this,MainActivity4::class.java))
        }else if(id == R.id.button4){
            startActivity(Intent(this,MainActivity5::class.java))
        }else if(id == R.id.button5){
            startActivity(Intent(this,MainActivity6::class.java))
        }
    }
}